packages/quick-edit-extension/src/cfs.ts

```diff
@@ -172,7 +172,11 @@ declare module "*.bin" {
 		);
 
 		for (const { path, contents } of files.files) {
-			const pathSegments = path.split("/");
+			const pathSegments = path
+				.split("/")
+				.filter(
+					(segment) => segment !== "." && segment !== ".." && segment !== ""
+				);
```

Comment:
Unfortunately not. This code runs in a web worker, and so `path` isn't available, and `Uri.parse()` isn't smart enough to figure this out. Module references shouldn't include `..`, since they won't (I think?) be evaluated at runtime.